src/Eval.cpp

```diff
@@ -195,8 +195,9 @@ void Eval::debug (bool value)
 std::vector <std::string> Eval::getOperators ()
 {
   std::vector <std::string> all;
-  for (unsigned int i = 0; i < NUM_OPERATORS; ++i)
-    all.push_back (operators[i].op);
+  all.reserve(sizeof(operators));
```

Comment:
This is not what we want, `sizeof(operators)` will return memory-size of the array (including elements), not the number of elements. Why not use the `NUM_OPERATORS` constant which was made for that?

Reply:
Hmm I'll look into this. Originally, I tried to use operators.size() but that obviously failed.

How about turning operators into an std::array?

Reply:
Actually std::array would allow turning this loop into a single std::transform. Can't do that as is since auto is not available in lambdas with C++11.